style.css

```diff
@@ -1,1875 +1,1058 @@
-  font-style: normal;
-  font-weight: 400;
-  font-size: 50px;
-  line-height: 100%;
-  color: #1a2236;
-  position: absolute;
+/* modal Project */
+.modal__header_title {
+  position: relative;
+  padding-top: 1.5em;
+  align-items: center;
 }
 
-.content p {
-  width: 90%;
-  height: 63px;
-  left: 16px;
-  top: 4305px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: 21px;
-  letter-spacing: -0.01em;
+.top_btn {
+  background: none;
+  border: none;
   color: #1a2236;
-  position: absolute;
+  font-size: 1.5rem;
+  margin-right: 1em;
+  margin-top: -2em;
 }
 
-form {
-  display: flex;
-  flex-direction: column;
+.modal__body_slide {
+  padding-top: 1.125em;
+}
+
+.slide_main {
+  position: relative;
+}
+
+.slide_items {
   justify-content: center;
 }
 
-label {
-  display: none;
+.slide_prev {
+  background: linear-gradient(90deg, rgba(129, 128, 128, 0.6811099439775911) 0%, rgba(255, 249, 249, 0) 100%);
 }
 
-#fullName {
-  position: absolute;
-  width: 90%;
-  height: 52px;
-  left: 16px;
-  top: 4400px;
-  background: #fff;
-  border: 1px solid #dfdfe0;
-  border-radius: 4px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 10px;
-  line-height: 15px;
-  color: #7d7c84;
-  padding-left: 15px;
-  padding-bottom: 20px;
+.slide_next {
+  right: 0;
 }
 
-#myEmail {
-  position: absolute;
-  width: 90%;
-  height: 52px;
-  left: 16px;
-  top: 4460px;
-  background: #fff;
-  border: 1px solid #1a2236;
-  border-radius: 4px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 10px;
-  line-height: 15px;
-  color: #7d7c84;
-  padding-left: 15px;
-  padding-bottom: 20px;
+.slide_item {
+  border-right: none;
+  flex: 1;
 }
 
-#msg {
-  position: absolute;
-  width: 90%;
-  height: 112px;
-  left: 16px;
-  top: 4520px;
-  background: #fff;
-  border: 1px solid #dfdfe0;
-  border-radius: 4px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 14px;
-  line-height: 21px;
-  color: #7d7c84;
-  padding-left: 15px;
-  padding-bottom: 20px;
+.slide_item:last-child {
+  border-right: 5px solid #e0e0e0;
 }
 
-#button {
-  position: absolute;
-  width: 90%;
-  height: 44px;
-  left: 16px;
-  top: 4652px;
-  background: #fff;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
-  text-align: center;
-  color: #1a2236;
-  border: 1px solid #1a2236;
-  cursor: pointer;
+.slide_item.active {
+  border: 5px solid #1a2236;
 }
 
-#button:hover {
-  background: #1a2236;
-  color: #fff;
-  transition: 1.5s;
+.modal__body_description {
+  padding-block: 2em;
 }
 
-.myFooter {
-  display: flex;
-  flex-direction: column;
+.modal__body_buttons button {
+  border: 2px solid #1a2236;
+  margin-bottom: 2em;
+  flex: 1;
+  color: #1a2236;
+  padding-left: 1em;
```

Comment:
[OPTIONAL] Kindly ensure that the button of the modal popup corresponds to that of the figma design since there are some differences.


 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-19 23-32-42](https://user-images.githubusercontent.com/65233860/202875852-5f1dfece-fe92-481d-ae17-c38f1332d5f2.png)  |  ![Screenshot from 2022-11-19 23-32-32](https://user-images.githubusercontent.com/65233860/202875874-e42b259b-952f-4806-a268-d21f2135e193.png)